### PR TITLE
Add integrations projects

### PR DESCRIPTION
**Pull request by gmoore**

### CODE REVIEW

File: challenges.md [hunk 1]
```diff
@@ -222,3 +222,32 @@ Candidates interested in becoming our first digital content strategist, we would
     - Tags for searchability and indexing
     - Please write the sample in [Markdown](https://en.wikipedia.org/wiki/Markdown)
 
+# Integrations Developer
+
+We have three projects to choose from if you're applying to be an integrations developer at Contrast.
+
+Pick a project that appeals to you and follow the instructions. Commit often with quality commit messages and so that we can see your thought process as the project progresses.
```
Comment (dhafley) [inline challenges.md]: I don't think we need 'and' in the second sentence.

File: challenges.md [hunk 2]
```diff
@@ -222,3 +222,32 @@ Candidates interested in becoming our first digital content strategist, we would
...
+
+After submitting your project, you'll have a conversation with one of our engineers about your code, the decisions you made, things you liked, things you didn't, etc...
```
Comment (dhafley) [inline challenges.md]: Should we include how to follow up when complete?  Might mean we do it for all projects no in this section?  Just an idea.

File: challenges.md [hunk 3]
```diff
@@ -222,3 +222,32 @@ Candidates interested in becoming our first digital content strategist, we would
...
+
+### Reverse Engineer An API Endpoint
+
+Clone the repository [here](https://github.com/Contrast-Security-OSS/project-reverse-engineer) and follow the instructions in the README.
+
+### Write a Contrast Security Command Line Client
+
+**Setup**
+
+* Sign up for a [Contrast Security Community Edition](https://www.contrastsecurity.com/contrast-community-edition) account.
```
Comment (dhafley) [inline challenges.md]: I would double check the approval process?  It might take 24-48 hours to get an account activated, for example.

Comment (gmoore) [reply challenges.md]: I got an activation email right away. Should be good.

File: challenges.md (hunk 1 above)
Comment (gmoore) [reply challenges.md]: 👍

**Pull request merged**